generator/subaru/subaru_global_2022.dbc

```diff
@@ -0,0 +1,14 @@
+CM_ "IMPORT subaru_global_2017.dbc";
+
+BO_ 292 ES_LKAS_2: 8 XXX
```

Comment:
```suggestion
BO_ 292 ES_LKAS_GEN2_5: 8 XXX # or _ALT?
```

neither are great names, but mark this as resolved if we don't yet know what is new about these cars. we can also make `_subaru_global` truly global (remove `ES_LKAS`) and put it into subaru_global_2017, so we can name this one `ES_LKAS` and require no changes in openpilot for the message name

Reply:
I like making _subaru_global truly global. I'll do that.

Reply:
You can do that cleanup later if you want

Reply:
I'll do _ALT for now. Once I get more info about other cars that may use this message, I'll revisit this.